### ISSUE TITLE
[9.0] Chrome always prints checkboxes empty (even when they are checked)

### ISSUE THREAD

**Issue by krueger80** [odoo/odoo#15147]
### Impacted versions:
9.0c
### Steps to reproduce:
1) Open Chrome browser, navigate to your instance of Odoo.
2) Go in Settings-->Users-->Groups
3) Open a group
4) Go on the Access Right tab
5) Print the current page

### Current behavior:
Only the document itself is printed witch is nice. But none of the checkboxes are checked, even if the group you selected had some access checked.

### Expected behavior:
We should see the checkboxes

### Video/Screenshot link (optional):
See the PDF I saved showing the issue.
[Other Extra Rights _ Portal - Odoo.pdf](https://github.com/odoo/odoo/files/715355/Other.Extra.Rights._.Portal.-.Odoo.pdf)

**Comment by Yenthe666** [comment]
@krueger80 on which version of Chrome is this? Tested on V47 without issues.

**Comment by krueger80** [comment]
@Yenthe666 I'm on Version 55.0.2883.87 m. Weird that you run a so old version of Chrome :) I tried also  version 42.0 of Firefox and it does the same.

**Comment by Yenthe666** [comment]
Hi @krueger80,

Just tested with 55.0.2883.87m. You're talking about this, right?
![image](https://cloud.githubusercontent.com/assets/6352350/22200719/9247ee30-e160-11e6-9373-8b6a23fe662e.png)
If this is the case then yes this doesn't work. @nim-odoo any idea?

**Comment by krueger80** [comment]
Yes preview and final print doesn't show the checks in the checkboxes.

**Comment by Yenthe666** [comment]
@nim-odoo and @mart-e could one of you look at it?

**Comment by mart-e** [comment]
It is due to [this rule](https://github.com/odoo/odoo/blob/732da30e20ae550648d1477ee4af7711b1be6cb7/addons/web/static/src/css/base.css#L2851-L2853) that is quite old (2012).

My guess is that it was added to hide the select box in the list view (the first row) which makes sense to be removed when printing.

@qsm-odoo can we just add one when the box is checked, like
```css
  .openerp .oe_list_content button, .openerp .oe_list_content input[type=checkbox]:checked {
    visibility: visible;
  }
```
(or better move it to print.less)

**Comment by qsm-odoo** [comment]
I don't know about these print rules but I suppose checking the checkbox status could be a valid fix yes. However in that case, I suppose you meant changing the mentioned rule to
```css
.openerp .oe_list_content button, .openerp .oe_list_content input[type=checkbox]:not(:checked) {
    visibility: hidden;
}
```
The file print.less you talk about is a bootstrap one though so we can't touch it. In enterprise and newer community version, these rules are gone as the DOM/rendering is different.

**Comment by mart-e** [comment]
oh yeah, that works too, I totally knew it :no_mouth: 
Fixed at d20fd7bdbb6a0a86dfbcbd996dd1f9b25c2a19d8
